app/views/questionnaire.py

```diff
@@ -109,6 +109,21 @@ def before_post_submission_request():
     )
 
 
+def _get_full_routing_path(schema):
+    sections = schema.get_sections()
+    return [path_finder.routing_path(section) for section in sections]
+
+
+@questionnaire_blueprint.route('/', methods=['GET'])
+@login_required
+@with_questionnaire_store
+@with_schema
+def get_questionnaire(schema, questionnaire_store):
+    router = Router(schema, completed_items=questionnaire_store.completed_items)
+    redirect_location = router.get_first_incomplete_location_in_survey()
+    return redirect(redirect_location.url())
```

Comment:
This ends up in a 500 sometimes as `router.get_first_incomplete_location_in_survey()` can return `None` and you can't call `.url` on `NoneType`.

Reply:
What scenario did you get the 500 for?

Reply:
Fixed in latest commit